pkg/rpc/kitex_gen/frontendservice/FrontendService.go

```diff
@@ -629,6 +629,7 @@ const (
 	TBinlogType_TRUNCATE_TABLE                   TBinlogType = 13
 	TBinlogType_RENAME_TABLE                     TBinlogType = 14
 	TBinlogType_RENAME_COLUMN                    TBinlogType = 15
+	TBinlogType_MODIFY_VIEW_DEF                  TBinlogType = 16
```

Comment:
16 -> 17

Reply:
Thank you for the rewives. I have already addressed this issue